daemon/cmdref.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2016-2018 Authors of Cilium
```

Comment:
wrong date?

Reply:
Similar to my other comment - I did this based off of when the code in this file was added into `daemon` - does the copyright only apply to when the file itself was created?

Reply:
That's why here I left it with as a question, I started wondering the same thing :thinking:

Reply:
See 
https://github.com/cilium/cilium/pull/6177#discussion_r231274398